<?xml version="1.0"?>
<!--PointCloud Transform-->
<launch>
	<node pkg="sensor_fusion" type="save_points_odom_2" name="save_points">
		<param name="/odom_parent_frame" type="string" value="/odom" />
		<param name="/odom_child_frame"  type="string" value="/base_link" />
		<remap from="cloud"		to="/cloud/tf" />
		<remap from="odom"		to="/odom" />
		<remap from="output"  	to="/cloud/lcl" />
    </node>
</launch>
